js/src/attribute-mapping/index.scss

```diff
@@ -0,0 +1,17 @@
+.gla-settings-attribute-mapping {
+
+	&__notice {
+		background: $gla-color-light-blue;
+		margin: 0;
+		margin-bottom: $grid-unit-20;
+
+		&.is-dismissible {
+			padding-right: 10px;
+		}
+	}
```

Comment:
💅
Looks like this CSS style and `$gla-color-light-blue` were missed to be removed when changing to the new design in https://github.com/woocommerce/google-listings-and-ads/pull/1662/commits/5d506d02bf6b45cfb303b0de47d0d91a7e6af36b?